content-src/components/DiscoveryStreamComponents/List/List.jsx

```diff
@@ -60,8 +60,16 @@ export function _List(props) {
 
   const recs = feed.data.recommendations;
 
-  let recMarkup = recs.slice(props.recStartingPoint, props.items).map((rec, index) => (
-    <ListItem {...rec} key={`ds-list-item-${index}`} index={index} type={props.type} dispatch={props.dispatch} />)
+  let recMarkup = recs.slice(0, props.items).map((rec, index) => (
+    <ListItem key={`ds-list-item-${index}`}
+      index={index}
+      type={props.type}
+      id={rec.id}
+      title={rec.title}
+      image_src={rec.image_src}
+      url={rec.url}
+      domain={rec.domain}
+      dispatch={props.dispatch} />)
```

Comment:
I think we'll need `rec.excerpt` for wide lists

Reply:
Keys on the rec seem to include ` "id", "url", "domain", "title", "excerpt", "image_src", "published_timestamp", "parameter_set", "domain_affinities", "item_score" `